source/include/FreeRTOS_DNS_Callback.h

```diff
@@ -29,17 +29,15 @@
 #ifndef FREERTOS_DNS_CALLBACK_H
 #define FREERTOS_DNS_CALLBACK_H
 
-/* FreeRTOS includes. */
-#include "FreeRTOS.h"
+/* Global Includes & Definitions. */
+#include "FreeRTOS_IP_Common.h"
```

Comment:
adding FreeRTOS_IP_Common to bring in FreeRTOS.h also adds dependencies upon many other modules.  This is a bad idea and encourages spaghetti code.